### PR TITLE
add: `gitui-git`

### PR DESCRIPTION
**Pull request by pythoncrazy**
Made a pacscript for gitui

### CODE REVIEW

File: packages/gitui-git/gitui-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,37 @@
+pkgname="gitui-git"
+gives="gitui"
+pkgdesc="Blazing fast terminal-ui for git written in Rust"
+url="https://github.com/extrawurst/gitui"
```
Comment (Elsie19) [inline packages/gitui-git/gitui-git.pacscript:4]:
```suggestion
url='https://github.com/extrawurst/gitui'
```

File: packages/gitui-git/gitui-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+source=("https://github.com/extrawurst/gitui.git")
+maintainer=("pythoncrazy <pinakpaliwal07@gmail.com>")
+pkgver="0.26.3"
+depends=('gcc' 'libgit2-dev' 'libxcb1' 'zlib1g-dev')
+makedepends=('git' 'cargo' 'python3') # xcb crate needs python
+external_connection="true"
+
+prepare() {
+  cd "${_archive}"
```
Comment (Elsie19) [inline packages/gitui-git/gitui-git.pacscript:13]: Do not use _archive, use something like `"${gives}"` or `"gitui"`.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/gitui-git/gitui-git.pacscript (hunk 2 above)
Comment (pythoncrazy) [reply packages/gitui-git/gitui-git.pacscript:13]: Alright, will do

**Comment by Elsie19** [comment]
Still waiting on _archive.

**Comment by Elsie19** [comment]
Debian stable fails with a versioning issue I think. Try to figure out the issue or just add it to `incompatible`.

**Comment by Elsie19** [comment]
Disable it on stable if it doesn't build.

**Comment by pythoncrazy** [comment]
I have disabled it on stable

File: packages/gitui-git/gitui-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,38 @@
+pkgname="gitui-git"
```
Comment (Elsie19) [inline packages/gitui-git/gitui-git.pacscript:1]: 7.

Comment (pythoncrazy) [reply packages/gitui-git/gitui-git.pacscript:1]: What is 7? Is this a reference to 8?

File: packages/gitui-git/gitui-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+gives="gitui"
+pkgdesc="Blazing fast terminal-ui for git written in Rust"
+url='https://github.com/extrawurst/gitui'
+source=("https://github.com/extrawurst/gitui.git")
+maintainer=("pythoncrazy <pinakpaliwal07@gmail.com>")
+pkgver="0.26.3"
+depends=('gcc' 'libgit2-dev' 'libxcb1' 'zlib1g-dev')
+makedepends=('git' 'cargo' 'python3')
```
Comment (oklopfer) [inline packages/gitui-git/gitui-git.pacscript:9]: cmake needed

**Review by oklopfer — CHANGES_REQUESTED** [review]

**Comment by oklopfer** [comment]
can you please generate srcinfo and add it thanks

File: packages/gitui-git/gitui-git.pacscript (hunk 4 above)
Comment (pythoncrazy) [reply packages/gitui-git/gitui-git.pacscript:9]: Yes sir, will do

**Comment by pythoncrazy** [comment]
Just added one.

**Review by oklopfer — APPROVED** [review]

**Pull request merged**